src/ensembl/src/content/app/custom-download/containers/content/result-holder/resultHolderHelper.ts

```diff
@@ -91,14 +106,14 @@ const flattenObject = (
       );
 };
 
-const formatResponseToArray = (responseData: any) => {
+const formatResponseToArray = (responseData: any): any => {
```

Comment:
Why are there so many anys here? Is this intentional?

Reply:
We do not have a proper structure of the result yet and this whole thing will be removed once we have the backend.